go/service/kbfs_mount.go

```diff
@@ -27,6 +31,35 @@ func (h *KBFSMountHandler) GetCurrentMountDir(ctx context.Context) (res string,
 	return h.G().Env.GetMountDir()
 }
 
+const waitForDirectMountTimeout = time.Second * 10
```

Comment:
Extremely minor, but I prefer doing `10 * time.Second` better, since it reads more like English.  And it's more consistent for grepping the code.